services/order.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Ticket, Order
+
+
+@transaction.atomic
+def create_order(
+    tickets: list[dict[str, int]],
+    username: str,
+    date: datetime = None
+) -> None:
+    user = get_user_model().objects.get(username=username)
```

Comment:
Using `get` without handling exceptions can lead to `DoesNotExist` errors if the username is not found. Consider handling this exception to provide a more graceful error handling.